### PR TITLE
Lab7

### PR DESCRIPTION
**Pull request by Dablup**
PR in my repo: https://github.com/Dablup/devops_labs/pull/5

### CODE REVIEW

**Comment by al1ych** [comment]
Все как надо и со скриншотами 👍

File: monitoring/LOGGING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ 1) Let's run our docker-compose.yml file to create needed containers
+ ![img.png](screenshots/img.png)
+As we can see, all containers started up successfully.
+
+
+ 2) Now let's check that application is accessible.
+ ![img_1.png](screenshots/img_1.png)![img_2.png](screenshots/img_2.png)
+
+
+ 3) Now, when our application is running, promtail container will read logs from the python app, send it to loki. In the same time grafana will take this logs from loki and show them to us.
```
Comment (SMore-Napi) [inline monitoring/LOGGING.md:12]: At the same time*

**Review by SMore-Napi — APPROVED** [review]
LGTM. Nice to see that you had listened to feedback from previous lab and provided a link to your PR Lab7 -> Lab6 in a description.

**Review by MikhailMarch — COMMENTED** [review]
Nice report

**Comment by emileyray** [comment]
LGTM

**Pull request closed without merging**